src/storage/storageController.ts

```diff
@@ -32,9 +32,12 @@ export type ConnectionsFromStorage = {
   [connectionId: string]: StoreConnectionInfo;
 };
 
+// Keytar is deprecated and no longer used. All new
+// connections use 'SecretStorage'.
```

Comment:
can't we just delete this?

Reply:
The other values here could still be on someone's stored connection if they open vscode for the first time in 5 months, I reckon we can remove these types but we should still account for when the `secretStorageLocation` isn't `SecretStorage`. Leaving these on the type I thought might give more insight into how it can be different. Do you prefer it deleted? I'm cool with either.